plugins/woocommerce/src/Blocks/BlockTypes/ProductFilters.php

```diff
@@ -36,6 +36,12 @@ protected function enqueue_data( array $attributes = array() ) {
 		$this->asset_data_registry->add( 'isProductArchive', is_shop() || is_product_taxonomy() );
 		$this->asset_data_registry->add( 'isSiteEditor', 'site-editor.php' === $pagenow );
 		$this->asset_data_registry->add( 'isWidgetEditor', 'widgets.php' === $pagenow || 'customize.php' === $pagenow );
+
+		$canonical_url_no_pagination = get_pagenum_link( 1 );
```

Comment:
```suggestion
		$canonical_url_no_pagination = get_pagenum_link();
```

Nitpick: Page number here is not really needed as it already defaults to 1 and is optional.

Reply:
I see. But in this case, I think explicitly passing `1` give us a clearer explaination that we're getting the URL fro the first page, without have to look at the definition of the `get_pagenum_link` function.